File: algorithms/242.cpp
#include <iostream>
#include <map>
using namespace std;

class Solution
{
public:
    bool isAnagram(string s, string t)
    {
        if ( s.size() != t.size() )
            return false;
        map<char, int> ss, tt;
        for ( unsigned int i = 0; i < s.size(); ++i )
            ss[s[i]]++, tt[t[i]]++;
        return ss == tt;
    }
};

int main()
{
    return 0;
}

File: algorithms/python/242.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        s = ''.join(sorted(list(s)))
        t = ''.join(sorted(list(t)))
        return s == t
